### PR TITLE
fix: remove start context

### PR DESCRIPTION
**Pull request by frdomovic**
# fix: remove start context

## Summary:

To unblock dev prev context creation and joining disabled from the admin dashboard.

### CODE REVIEW

File: src/components/context/ContextTable.tsx [hunk 1]
```diff
@@ -33,8 +33,6 @@ interface ContextTableProps {
 
 export default function ContextTable({
   nodeContextList,
-  navigateToStartContext,
-  navigateToJoinContext,
```
Comment (MatejVukosav) [inline src/components/context/ContextTable.tsx:37]: We still have join context

**Comment by MatejVukosav** [comment]
You can just hide the button, as we will return it in a few weeks. 

Can you reflect the change in documentation?

**Comment by frdomovic** [comment]
> You can just hide the button, as we will return it in a few weeks.
> 
> Can you reflect the change in documentation?

To hide this button it means to remove params fields from the header component - which is done. Then you get unused vars error so also removed that to not cause issues.
Lastly added redirect if somebody navigates to this page manually. 
Will update docs

**Comment by frdomovic** [comment]
@MatejVukosav  https://github.com/calimero-network/calimero-network.github.io/pull/14

**Review by MatejVukosav — APPROVED** [review]
Update description

**Pull request merged**